content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -190,13 +190,12 @@ PRISMA_FMT_BINARY=custom/my-custom-format-engine-unix
 
 #### <inlinecode>PRISMA_CLI_BINARY_TARGETS</inlinecode>
 
-`PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that the CLI will download, it must be provided during `npm install`.
+`PRISMA_CLI_BINARY_TARGETS` can be used to specify one or more binary targets that Prisma CLI will download during installation (so it must be provided during `npm install` of Prisma CLI and does not work affect runtime of Prisma CLI or Prisma Client at all).
 
-For example:
-If you are on macOS, the binary target is `darwin` & AWS Lambda target is `rhel-openssl-1.0.x`
+This is to be used if you deploy to a specific platform via an upload of a local project including dependencies and your local environment is different from the target (e.g. AWS Lambda is `rhel-openssl-1.0.x`, and your local environment might be macOS `darwin`) and makes sure the target engine files are also downloaded.
 
```

Comment:
Consider breaking into two sentences: 
"Use `PRISMA_CLI_BINARY_TARGETS` if you 1) deploy to a specific platform via an upload of a local project that includes dependencies, and 2) your local environment is different from the target (e.g. AWS Lambda is `rhel-openssl-1.0.x`, and your local environment might be macOS `darwin`). Using the `PRISMA_CLI_BINARY_TARGETS` environment variable ensures that the target engine files are also downloaded."